packages/homebrew/layer0.rb

```diff
@@ -0,0 +1,22 @@
+class Layer0 < Formula
+  desc "Framework that helps you deploy to AWS with minimal fuss"
+  homepage "http://layer0.ims.io"
+  url "https://github.com/quintilesims/layer0/archive/v0.10.6.tar.gz"
```

Comment:
we should use latest version of Layer0

Reply:
btw, is there any way to have these scripts always grab `latest`, or do we always have to manually update them?

Reply:
It doesn't look like it. We would probably have to do a script
https://stackoverflow.com/questions/24987542/is-there-a-link-to-github-for-downloading-a-file-in-the-latest-release-of-a-repo